airflow/providers/databricks/operators/databricks_sql.py

```diff
@@ -163,18 +163,23 @@ class DatabricksCopyIntoOperator(BaseOperator):
         or ``sql_endpoint_name`` must be specified.
     :param sql_endpoint_name: Optional name of Databricks SQL Endpoint.
         If not specified, ``http_path`` must be provided as described above.
-    :param files: optional list of files to import. Can't be specified together with ``pattern``.
+    :param files: optional list of files to import. Can't be specified together with ``pattern``. (templated)
     :param pattern: optional regex string to match file names to import.
         Can't be specified together with ``files``.
     :param expression_list: optional string that will be used in the ``SELECT`` expression.
+    :param credential: optional credential configuration for authentication against a specified location.
```

Comment:
It would be great to provide a link to the docs explaining what are possible credential/encryption options. And I think  example_dags should show a few examples (not only unit tests)

Reply:
(the reason is that it creates a custom query and it's not entirely obvious that it is happening and how the query is constructed).

Reply:
yes, it's good idea - let me add that information

Reply:
done